core/port.cc

```diff
@@ -175,13 +175,25 @@ Port::PortStats Port::GetPortStats() {
     ret.inc.packets += inc.packets;
     ret.inc.dropped += inc.dropped;
     ret.inc.bytes += inc.bytes;
+    for (size_t i = 0; i < inc.requested_hist.max_size(); i++) {
```

Comment:
😿 
Sorry for the scrutiny (I promise this is the last). Here `max_size()` should be `size()`. 
To remove the same-size assumption and reduce verbosity a bit, how about this?
```
struct BatchHistogram: public std::array<uint64_t, bess::PacketBatch::kMaxBurst + 1> {
  BatchHistogram &operator+=(BatchHistogram &rhs) {
    for (size_t i = 0; i < size(); i++) {
        (*this)[i] += rhs[i];
    }
  }
};

struct QueueStats {
    ...
    BatchHistogram requested_hist;
    ...
};

ret.inc.requested_hist += inc.requested_hist;
```

Reply:
wow that's like magic

Reply:
will do!

Reply:
Thou shalt clang-format